src/jdk.httpserver/share/classes/com/sun/net/httpserver/Filter.java

```diff
@@ -197,41 +195,38 @@ public String description() {
      * Returns a post-processing {@code Filter} with the given description and
      * operation.
      *
-     * <p>The {@code description} string describes the returned filter. The
-     * {@link Consumer operation} is the effective implementation of the filter.
-     * It is executed for each {@code HttpExchange} after invoking either the
-     * next filter in the chain or the exchange handler (if this filter is the
-     * final filter in the chain).
+     * <p>The {@link #description() description} describes the returned filter.
+     * The {@link Consumer operation} is the effective implementation of the
+     * filter. It is executed for each {@code HttpExchange} after invoking
+     * either the next filter in the chain or the exchange handler (if this
+     * filter is the final filter in the chain).
      *
      * @apiNote
      * An afterHandler filter is typically used to examine the exchange state
-     * rather than modifying it. The filter operation is executed after
+     * rather than modifying it. The filter {@code operation} is executed after
      * {@link Filter.Chain#doFilter(HttpExchange)} is invoked, this means any
      * subsequent filters in the chain and the exchange handler have been
-     * executed and the response has commonly been sent. The filter is only
-     * expected to send the response in the uncommon case that the exchange
-     * has not been handled before the filter is executed.
+     * executed. The filter {@code operation} is not expected to handle the
+     * exchange or {@linkplain HttpExchange#sendResponseHeaders(int, long) send the response headers}.
+     * Doing so is likely to fail, since this is commonly done by the exchange
+     * handler.
      *
-     * <p> Example of logging the response code of all exchanges:
+     * <p> Example of adding a filter that logs the response code of all exchanges:
      * <pre>{@code
-     *     var filter = Filter.afterHandler("Log response code", (var e) -> log(e.getResponseCode());
-     *     var server = HttpServer.create(new InetSocketAddress(0), 10);
-     *     var context = server.createContext("/", new SomeHandler());
-     *     context.getFilters().add(filter);
-     *     server.start();
+     *     var filter = Filter.afterHandler("Log response code", e -> log(e.getResponseCode());
+     *     httpContext.getFilters().add(filter);
      * }</pre>
      *
-     * <p> Example of adding a sequence of afterHandler filters to a context
-     * (note the ordering):
+     * <p> Example of adding a sequence of afterHandler filters to a context:<br>
+     * The order in which the filters are invoked is reverse to the order in
+     * which they are added to the context's filter-list.
```

Comment:
May need to be a bit more precise. The filters are invoked in the order they are added, but their operations are invoked in the reverse order. I'd suggest:

```
     * The order in which the filter's operations are invoked is reverse to the order in
     * which the filters are added to the context's filter-list.
```